relnotes/verify.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to Sociomantic assert/enforce policies.
```

Comment:
Maybe in https://github.com/sociomantic-tsunami/sociomantic or https://github.com/sociomantic-tsunami/tsunami?